pulp_rpm/app/viewsets.py

```diff
@@ -115,19 +116,21 @@ def sync(self, request, pk):
         Dispatches a sync task.
         """
         repository = self.get_object()
-        serializer = RepositorySyncURLSerializer(
+        serializer = RpmRepositorySyncURLSerializer(
             data=request.data,
             context={'request': request}
         )
         serializer.is_valid(raise_exception=True)
         remote = serializer.validated_data.get('remote')
+        to_skip = serializer.validated_data.get('skip')
```

Comment:
Is there a reason to call it `to_skip` when everywhere else it's `skip`? :) just curious

Reply:
good point, renamed to skip for consistency

Reply:
what about ```skip_types``` which is more explicit?

Reply:
fine as well, I'm all for any consistency